### ISSUE TITLE
Incomplete code

### ISSUE THREAD

**Issue by Tara-Liu** [JixuanLiang/deblur-LGP#1]
Which file should I run to test my image, it seems that I lack some programs

**Comment by Tara-Liu** [comment]
The function or variable 'LGP gen Patchsum n' is not recognized

**Comment by Redamancy0222** [comment]
Hello, the output variables in this line are not necessary. You can simply comment out this line of code and run it.

**Comment by JixuanLiang** [comment]
> Hello, the output variables in this line are not necessary. You can simply comment out this line of code and run it.

True, it's my mistake, i have committed the change. Thanks for your answer.

This project only provides programs on the Kohler dataset for reference by other researchers working on deblurring algorithms. If you want to test your own images, you can refer to demo_LGP_kohler.m and func_lgp_demo.m to fill in the relevant parameters. @Tara-Liu
